### DISCUSSION TITLE
欢迎来到Web3wiki 项目讨论区

### DISCUSSION THREAD

**Discussion by yanboishere** [Web3-Club/Web3wiki-Website#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JiceJin** [comment]
###  hi , i'm JiceJin, i am glad to contribute our community together.

**Comment by Jface001** [comment]
hi , I'm Jface, I'm glad to contribute our community together.
